<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
	background-color:transparent;
/*
	background-color: rgba(28, 225, 255, 100);
	border-top-left-radius:80px;
	border-bottom-left-radius:80px;
*/
}
	
</string>
   </property>
   <widget class="QTabWidget" name="tw_connect">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>211</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>150</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Blogger Sans Light</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTabWidget::pane {
background-color: green;
left:5px;
right:5px;
bottom:5px;
}

QTabWidget QTabBar::tab {
border-color: black;
border-width: 3px;
border-bottom:none;
border-top-left-radius: 6px;
border-top-right-radius: 6px;
background:transparent;
color:#EEEEEE;
min-width:25ex;
min-height:10ex;
}

QTabWidge QTabBar::tab:!selected:hover{
width:100px;
height:24px;
background:rgb(255, 255, 255, 100);
border-color: black;
}

QTabWidget QTabBar::tab:selected{
background:#00ADB5;
color:#222831;
}

QTabWidget QTabBar::tab:!selected{
background:none;
color:#EEEEEE;
border:none;
}
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="lab_background_2">
      <property name="geometry">
       <rect>
        <x>-140</x>
        <y>20</y>
        <width>350</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(47, 47, 47, 230);
/*
border-top-right-radius:40;
border-bottom-right-radius:40;
*/</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="bn_clearlog">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>720</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_log">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>694</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graphicsView_charts">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>10</y>
      <width>351</width>
      <height>398</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bn_chart_stop">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>680</y>
      <width>75</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="motordial" name="dial">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> border:2px solid rgb(0,0,0);</string>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_velocity">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>80</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color: rgb(9, 202, 252);
	border:2px solid rgb(9, 202, 252);
	background-color:rgb(50, 50, 50, 0);
	padding: 0px 0px 0px 10px;
}
</string>
    </property>
    <property name="text">
     <string>12.4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_angle">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>130</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color: rgb(9, 202, 252);
	border:2px solid rgb(9, 202, 252);
	background-color:rgb(50, 50, 50, 0);
	padding: 0px 0px 0px 10px;
}
</string>
    </property>
    <property name="text">
     <string>550.21</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_velocity_2">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>30</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color: rgb(9, 202, 252);
	border:2px solid rgb(9, 202, 252);
	background-color:rgb(50, 50, 50, 0);
	padding: 0px 0px 0px 10px;
}
</string>
    </property>
    <property name="text">
     <string>12.4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid  rgb(9, 202, 252);
color: rgb(9, 202, 252);</string>
    </property>
    <property name="text">
     <string>Angle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_background">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>10</y>
      <width>831</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(47, 47, 47, 210);
border-top-left-radius:40;
border-bottom-left-radius:40;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>59</y>
      <width>340</width>
      <height>340</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:170;
background-color: rgb(60, 60, 60);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bn_close">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>30</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px solid rgb(9, 202, 252);
	background-color: rgb(9, 202, 252, 20);
	color:rgb(217, 217, 217);
	border-top-left-radius:20;
	
	padding:10px;
	image: url(:/icons/icons/close.png);
}

QPushButton:hover{

	border:2px solid #53DCFF;
	background-color: rgb(9, 202, 252, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>90</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px solid rgb(9, 202, 252);
	background-color: rgb(9, 202, 252, 20);
	color:rgb(217, 217, 217);
	padding:10px;
	
	
	
	image: url(:/icons/icons/logIcon.png);
}

QPushButton:hover{

	border:2px solid #53DCFF;
	background-color: rgb(9, 202, 252, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>150</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px solid rgb(9, 202, 252);
	background-color: rgb(9, 202, 252, 20);
	color:rgb(217, 217, 217);
	padding:10px;
	
	
	image: url(:/icons/icons/splineIcon.png);
}

QPushButton:hover{

	border:2px solid #53DCFF;
	background-color: rgb(9, 202, 252, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>290</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px solid rgb(9, 202, 252);
	background-color: rgb(9, 202, 252, 20);
	color:rgb(217, 217, 217);
	border-bottom-left-radius:20;
	padding:10px;
	
	
	image: url(:/icons/icons/setting.png);
}

QPushButton:hover{

	border:2px solid #53DCFF;
	background-color: rgb(9, 202, 252, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>333</x>
      <y>190</y>
      <width>701</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/icons/bk_img.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>809</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px solid rgb(9, 202, 252);
	background-color: rgb(9, 202, 252, 20);
	color:rgb(217, 217, 217);
	padding:10px;
	
	
	image: url(:/icons/icons/splineIcon.png);
}

QPushButton:hover{

	border:2px solid #53DCFF;
	background-color: rgb(9, 202, 252, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>809</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px solid rgb(9, 202, 252);
	background-color: rgb(9, 202, 252, 20);
	color:rgb(217, 217, 217);
	padding:10px;
	
	
	image: url(:/icons/icons/splineIcon.png);
}

QPushButton:hover{

	border:2px solid #53DCFF;
	background-color: rgb(9, 202, 252, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>809</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px solid rgb(9, 202, 252);
	background-color: rgb(9, 202, 252, 20);
	color:rgb(217, 217, 217);
	padding:10px;
	
	
	image: url(:/icons/icons/splineIcon.png);
}

QPushButton:hover{

	border:2px solid #53DCFF;
	background-color: rgb(9, 202, 252, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid  rgb(9, 202, 252);
color: rgb(9, 202, 252);</string>
    </property>
    <property name="text">
     <string>Velocity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid  rgb(9, 202, 252);
color: rgb(9, 202, 252);</string>
    </property>
    <property name="text">
     <string>Accelerator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="connectWidget" name="wdt_connect" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border:2px solid rgb(9, 202, 252);
	background-color: rgb(9, 202, 252, 20);
	color:rgb(217, 217, 217);
	padding:12px;
	image: url(:/icons/icons/connection.png);</string>
    </property>
   </widget>
   <zorder>lab_background</zorder>
   <zorder>label_13</zorder>
   <zorder>tw_connect</zorder>
   <zorder>bn_clearlog</zorder>
   <zorder>te_log</zorder>
   <zorder>graphicsView_charts</zorder>
   <zorder>bn_chart_stop</zorder>
   <zorder>dial</zorder>
   <zorder>le_velocity</zorder>
   <zorder>le_angle</zorder>
   <zorder>le_velocity_2</zorder>
   <zorder>label_5</zorder>
   <zorder>bn_close</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>wdt_connect</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>motordial</class>
   <extends>QDial</extends>
   <header>motordial.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QGraphicsView</extends>
   <header>qcustomplot.h</header>
  </customwidget>
  <customwidget>
   <class>connectWidget</class>
   <extends>QWidget</extends>
   <header>connectwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
